src/SFA.DAS.AdminService.Settings/IWebConfiguration.cs

```diff
@@ -29,5 +29,7 @@ public interface IWebConfiguration
         /// Gets or Sets the UseDfESignIn property.
         /// </summary>
         bool UseDfESignIn { get; }
```

Comment:
This comment can be removed since the property is self-explanatory. Also it doesn't set the property